CMakeLists.txt

```diff
@@ -91,6 +91,9 @@ if(SFML_BUILD_WINDOW)
     sfml_set_option(SFML_OPENGL_ES ${OPENGL_ES} BOOL "TRUE to use an OpenGL ES implementation, FALSE to use a desktop OpenGL implementation")
 endif()
 
+# add an option for SFML compatibility, temporarily
+sfml_set_option(MML_SFML_COMPAT FALSE BOOL "TRUE to include API alterations allowing compatibility with SFML")
```

Comment:
docstring should better be "Include API alterations allowing compability with SFML"

People will get that "TRUE" means "ON" :)

Reply:
Ok :)

Reply:
Ok :)